doc/releases/changelog-0.28.0.md

```diff
@@ -611,16 +436,40 @@
 * Updated `qml.transforms.metric_tensor`, `qml.transforms.adjoint_metric_tensor`, `qml.qinfo.classical_fisher`, and `qml.qinfo.quantum_fisher` to support the new return types.
```

Comment:
Could we move these smaller points down to improvements?

Reply:
Sure!